src/main/java/core/basesyntax/BooleanExpression.java

```diff
@@ -14,6 +14,7 @@
      * без использования конструкций if-else</p>
      */
     public boolean booleanExpression(boolean a, boolean b, boolean c, boolean d) {
-        return false;
+        return (a && b && !c && !d) || (!a && !b && c && d) || (!a && b && c && !d)
+                || (a && !b && !c && d) || (a && !b && c && !d) || (!a && b && !c && d);
```

Comment:
try to simplify it with binary operators

Reply:
do you mean that I can make the second part look like the first one, but using "!"?